tests/Unit/CMClientPushSettings.tests.ps1

```diff
@@ -0,0 +1,635 @@
+                                PropertyName = 'Filters'
+                                Value        = 4
+                            }
+                            @{
+                                PropertyName = 'Settings'
+                                Value1       = 'Inactive'
+                            }
+                            @{
+                                PropertyName = 'AutoInstallSiteSystem'
+                                Value        = 0
+                            }
+                        )
+                    }
+                )
+
+                $siteComponent5 = @(
+                    @{
+                        ComponentName = 'SMS_Discovery_Data_Manager'
+                        Props         = @(
+                            @{
+                                PropertyName = 'Filters'
+                                Value        = 5
+                            }
+                            @{
+                                PropertyName = 'Settings'
+                                Value1       = 'Inactive'
+                            }
+                            @{
+                                PropertyName = 'AutoInstallSiteSystem'
+                                Value        = 0
+                            }
+                        )
+                    }
+                )
+
+                $siteComponent6 = @(
+                    @{
+                        ComponentName = 'SMS_Discovery_Data_Manager'
+                        Props         = @(
+                            @{
+                                PropertyName = 'Filters'
+                                Value        = 6
+                            }
+                            @{
+                                PropertyName = 'Settings'
+                                Value1       = 'Inactive'
+                            }
+                            @{
+                                PropertyName = 'AutoInstallSiteSystem'
+                                Value        = 0
+                            }
+                        )
+                    }
+                )
+
+                $siteComponent7 = @(
+                    @{
+                        ComponentName = 'SMS_Discovery_Data_Manager'
+                        Props         = @(
+                            @{
+                                PropertyName = 'Filters'
+                                Value        = 7
+                            }
+                            @{
+                                PropertyName = 'Settings'
+                                Value1       = 'Active'
+                            }
+                            @{
+                                PropertyName = 'AutoInstallSiteSystem'
+                                Value        = 1
+                            }
+                        )
+                    }
+                )
+
+                $clientPushProps = @{
+                    Props     = @(
```

Comment:
it looks like the = @( was lined up with the = above it everywhere else EX: line 186. Match here for standardization

Reply:
Props = lines up with PropLists = lining up items inside of the hash table